app/connectors/SecureMessageFrontendConnector.scala

```diff
@@ -39,37 +38,33 @@ class SecureMessageFrontendConnector @Inject()(httpClient: HttpClient, config: A
       conversationEndpointQueryParams
     ).map(response => ConversationPartial(response.body))
 
-  def retrieveReplyResult(client: String, conversationId: String)(implicit hc: HeaderCarrier): Future[ReplyResultPartial] =
-    SecureMessageFrontendConnector.fakeReplyResultPartial
-  /*
-    fetchPartial(
-      config.microservice.services.secureMessaging.replyResultEndpoint(client, conversationId),
-      s"the result of replying to '$client/$conversationId' conversation"
-    )
-    .map(response => ReplyResultPartial(response.body))
-   */
-
-  def submitReply(client: String, conversationId: String, reply: Map[String, Seq[String]])(implicit hc: HeaderCarrier): Future[Unit] =
-    Future.successful(())
-  /*
+  def submitReply(client: String, conversationId: String, reply: Map[String, Seq[String]])(
+    implicit hc: HeaderCarrier
+  ): Future[Option[ConversationPartial]] =
     httpClient
-      .doPost(
-        config.microservice.services.secureMessaging.submitReplyEndpoint(client, conversationId),
-        Json.toJson(reply)
-      )
+      .POSTForm(config.microservice.services.secureMessaging.submitReplyEndpoint(client, conversationId), reply)
       .flatMap { response =>
+        logger.warn(s"Submitting reply form to secure-messaging-frontend returned the status ${response.status} ${response.body}")
```

Comment:
As a warning?

Reply:
Ah yeah good spot. Was just for debugging the DC call. I've removed this line now